### ISSUE TITLE
SC2096 (shebang multiple arguments) warning incorrectly spans the entire file

### ISSUE THREAD

**Issue by lilyball** [koalaman/shellcheck#1620]
The rule to detect multiple args to a shebang flags the entire file as an error instead just flagging the shebang. This is extremely irritating in IDEs that display the entire error span like VSCode.

#### For bugs
- Rule Id (if any, e.g. SC1000): SC2096
- My shellcheck version (`shellcheck --version` or "online"): 0.6.0
- [X] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [ ] I tried on shellcheck.net and verified that this is still a problem on the latest commit (not applicable, shellcheck.net won't show json output)

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash -x -e
echo hi
```

#### Here's what shellcheck currently says:

```json
[{"file":"foo.sh","line":1,"endLine":3,"column":1,"endColumn":1,"level":"error","code":2096,"message":"On most OS, shebangs can only specify a single parameter."}]
```

#### Here's what I wanted or expected to see:

```json
[{"file":"foo.sh","line":1,"endLine":1,"column":1,"endColumn":18,"level":"error","code":2096,"message":"On most OS, shebangs can only specify a single parameter."}]
```

Or better yet, only flag the extra args:

```json
[{"file":"foo.sh","line":1,"endLine":1,"column":16,"endColumn":18,"level":"error","code":2096,"message":"On most OS, shebangs can only specify a single parameter."}]
```

**Comment by koalaman** [comment]
Wow, that's annoying! This has now been fixed, thanks!
